api/spec/lib/spree/api_configuration_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe Spree::ApiConfiguration do
+  subject(:config) { Spree::ApiConfiguration.new }
+
+  describe "#promotion_attributes" do
+    subject(:promotion_attributes) { config.promotion_attributes }
+
+    it { is_expected.to eq(Spree::Config.promotions.promotion_api_attributes) }
+  end
+
+  describe "#promotion_attributes=" do
```

Comment:
I think this also supports pushing new attributes to the array. 

```ruby
config.promotion_attributes << [:name]

# or

config.promotion_attributes.push(:name)
```

And also removing items from the array. Shouldn't we also support those kind of customizations?

Reply:
All of these are still possible, as the list of attributes is just an array of symbols at the end. I've added some specs to prove it.